site/content/docs/5.3/examples/e-shop/index.html

```diff
@@ -0,0 +1,1567 @@
+                </p>
+                <span class="visually-hidden">Still <span>1</span> hour<span></span> <span>11</span> minute<span>s</span> and <span>27</span> second<span>s</span> to benefit from this offer.</span>
+              </div>
+            </div>
+            <div class="card-body d-flex flex-column align-items-stretch justify-content-between p-0">
+              <div class="row mb-3 align-items-start">
+                <div class="col-5 small fw-bold">
+                  <div class="mb-1 ratio ratio-3x4">
+                    {{< placeholder width="100%" height="160" text="icon" >}}
+                  </div>
+                  DAS: 0,99 W/Kg
+                </div>
+                <div class="col-7">
+                  <p class="fs-4 fw-bold mb-2" aria-hidden="true">xxxxx xxxxxx xx xxx xxx</p>
+                  <fieldset class="star-rating star-rating-sm pt-md-1 mb-3" disabled aria-hidden="true">
+                    <legend class="visually-hidden">rated 3 stars out of 5</legend>
+                    <input type="radio" id="terrible1" name="rating1" value="1" class="visually-hidden">
+                    <label for="terrible1" title="Terrible"></label>
+
+                    <input type="radio" id="bad1" name="rating1" value="2" class="visually-hidden">
+                    <label for="bad1" title="Bad"></label>
+
+                    <input type="radio" id="mixed1" name="rating1" value="3" class="visually-hidden" checked>
+                    <label for="mixed1" title="Mixed"></label>
+
+                    <input type="radio" id="good1" name="rating1" value="4" class="visually-hidden">
+                    <label for="good1" title="Good"></label>
+
+                    <input type="radio" id="excellent1" name="rating1" value="5" class="visually-hidden">
+                    <label for="excellent1" title="Excellent"></label>
+                  </fieldset>
+                  <span class="visually-hidden">rated 3 stars out of 5</span>
+                  <p class="mb-3 fw-bold d-inline-flex align-middle">
+                    <svg class="border border-1 rounded-circle me-2" width="1.25rem" height="1.25rem" viewBox="0 0 20 20" version="1.1" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false">
+                      <defs>
+                        <linearGradient id="Gradient1">
+                          <stop offset="0%" stop-color="#0400ff"/>
+                          <stop offset="100%" stop-color="#fff800"/>
+                        </linearGradient>
+                      </defs>
+                      <circle cx="10" cy="10" r="10" fill="url('#Gradient1')"/>
+                    </svg>
+                    5 colors
+                  </p>
+                  <p class="mb-0 pt-md-1">
+                    <span class="d-block mb-1 fw-bold">From</span>
+                    <span class="fs-4 fw-bold text-primary">2 500 350 FCFA</span><br>
+                    <span class="visually-hidden">instead of </span>
```

Comment:
Looks ok according to https://a11y-guidelines.orange.com/en/articles/price-vocalization/
It may be possible to improve it, we should double check with our a11y expert.